.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+    "trailingComma": "es5",
+    "printWidth": 80,
+    "tabWidth": 2,
```

Comment:
These are defaults is there a reason of explicitly defining them?

Reply:
No reason other than just being explicit, when I created the file I tried my best to describe the code base as it exists today.